.github/std-unexploded.yml

```
@@ -0,0 +1,107 @@
+# yq '. | explode(.)' std-unexploded.yml > workflows/std.yml
+name: CI/CD
+
+on:
+  pull_request:
```

```suggestion
  workflow_trigger:
  pull_request:
```
This will likely to come in handy (please double check if that's how workflow trigger event calls)